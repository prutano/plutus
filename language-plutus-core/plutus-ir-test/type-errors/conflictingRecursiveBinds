(lam x (con integer)
(let
  (rec)
    (termbind (strict) (vardecl j (con integer)) (con 3))
    (termbind (strict) (vardecl j (con integer)) (con 3))
    (con 5)
    )
  )

