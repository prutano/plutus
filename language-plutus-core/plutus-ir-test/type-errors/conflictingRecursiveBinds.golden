Error during typechecking:
Some binds of a recursive-let conflict with each other at conflictingRecursiveBinds:2:2
Conflicting identifier: j